app/controllers/sessions_controller.rb

```diff
@@ -1,14 +1,17 @@
 class SessionsController < Devise::SessionsController
-    before_action :authenticate_user!
     respond_to :json
-  
-    private
-  
+    before_action :rewrite_param_names, only: [:create]
+    def create
+      super
+    end
```

Comment:
leave an empty line between the end of a method and the beginning of the next one